usr.sbin/bsdinstall/scripts/hostname

```diff
@@ -40,6 +40,9 @@ f_include $BSDCFG_SHARE/dialog.subr
 # Default value
 #
 : ${HOSTNAME=$( hostname )}
+if [ "X$HOSTNAME" = "X" ]; then
```

Comment:
This is fine, though I feel like `if [ -z "$HOSTNAME" ];` is how I see this tested nowadays.

Reply:
Or even as is done in ports rc.d scripts where default values aren't in /etc/defaults:

```
: ${HOSTNAME="cheribsd"}
```

Reply:
You'd want := in this case; = is if unset, := is if unset or empty, and the line above means it'll be empty, not unset

Reply:
Oh yes, the sh(1) manpage doesn't even document the '=' variant, just ':='.

Reply:
I’m afraid my theory was that I’d just adopt similar structure to nearby instances of similar things. But I think the net conclusion of this discussion is that this bit is fine to go, so will mark as resolved.